### DISCUSSION TITLE
Timeline for 3.7?

### DISCUSSION THREAD

**Discussion by vdellamea** [jamulussoftware/jamulus#1196] [General]
Hi great Jamulus developers, 
do you have a timeline for the 3.7 release? I am going to update Jamulus Recording Remote, and I would wait for that, to be sure everything is okay. Thanks.

**Comment by hoffie** [comment]
Hi!

thanks for asking! :) No specific timeline, but Release Candidate 1 can hopefully be tagged really soon. If no issues come up, this will be tagged as 3.7.0 final.

You can track progress of the remaining tasks here: https://github.com/jamulussoftware/jamulus/milestone/1

Kind regards,
Christian
